romana-install/romana-setup

```diff
@@ -147,6 +147,7 @@ if (( $# > 0 )); then
 				shift
 				;;
 			--core-branch)
+			        core_source="github"
```

Comment:
Greg wanted that option to implicitly mean `--source` too. (Builds from source instead of downloaded binaries.)